Emulator/pal/gc.asm

```diff
@@ -10,15 +10,13 @@ TCHR,	0000					/TEMP CHAR
 
 		*200					/LOCATE @ 0200
 
-		CLA						/CLEAR ACC
-		CLL						/CLEAR LINK
+		CLA CLL					/CLEAR ACC, CLEAR LINK
 		TAD MSG					/LOAD ADDR OF MESSAGE
 		DCA AIX1				/DUMP IN AUTOINDEX REGISTER #1
 
-		CLA						/CLEAR ACC
-		CLL						/CLEAR LINK
+		CLA CLL					/CLEAR ACC, CLEAR LINK
 
-LOOP,	NOP						/READ AND PRINT LOOP
+LOOP,							/READ AND PRINT LOOP
```

Comment:
NOP is an unnecessary instruction here. Sins you don't call LOOP with JMS there is no need to reserve space. The label "LOOP" will refer to the next line.

Reply:
I agree! However a NOP here won't reserve extra space, just init it to a value != 0.